recipes/wilzegers_autotest/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+requires_conan_version = ">=1.33.0"
+
+
+class WilzegersAutotestConan(ConanFile):
+    name = "wilzegers_autotest"
```

Comment:
You can name it `wilzigers-autotest`.
Imho, that's nicer to read.
We have already used dashes to separate users from project names.

Reply:
ready